sms-notifications/assets/index.js

```diff
@@ -69,6 +79,9 @@ function sendMessages(form) {
 
 sendNotificationForm.addEventListener('submit', (evt) => {
   evt.preventDefault();
+
+  addRecipientFromInput();
```

Comment:
I think we should have this behavior only if no number was added to the list before. If there is one in the list the person probably realized how the page actually works at which point this behavior would be counter-intuitive.

Reply:
That sounds good to me -- I incorporated this change into the amended commit.